app/models/miq_server.rb

```diff
@@ -482,11 +482,7 @@ def restart
     _log.info("Server restart initiating...")
     self.update_attribute(:status, "restarting")
 
-    logfile = Rails.root.join("log/vmdb_restart.log")
-    File.delete(logfile) if File.exist?(logfile)
-
-    pid = spawn("nohup #{RESTART_SCRIPT}", [:out, :err] => [logfile, "a"])
```

Comment:
yea, I was wondering if freezing that string would save this. but didn't push that PR yet. looks like it is not necessary

otoh, this warning is what is pushing me back into awesome_spawn#detach land